src/qml/controls/OnboardingInfo.qml

```diff
@@ -30,55 +30,64 @@ Item {
 
     implicitWidth: 600
 
-    ColumnLayout {
-        anchors.horizontalCenter: parent.horizontalCenter
+    ScrollView {
         width: parent.width
-        id: information
-        spacing: 0
-        Loader {
-            id: banner_loader
-            active: root.bannerActive
-            visible: active
-            Layout.alignment: Qt.AlignCenter
-            Layout.topMargin: root.bannerMargin
-            sourceComponent: root.bannerItem
-        }
-        Header {
-            Layout.fillWidth: true
-            Layout.leftMargin: 20
-            Layout.rightMargin: 20
-            bold: root.bold
-            center: root.center
-            header: root.header
-            headerMargin: root.headerMargin
-            headerSize: root.headerSize
-            description: root.description
-            descriptionMargin: root.descriptionMargin
-            descriptionSize: root.descriptionSize
-            subtext: root.subtext
-            subtextMargin: root.subtextMargin
-            subtextSize: root.subtextSize
+        height: parent.height
+        clip: true
+        contentWidth: width
+        contentHeight: information.height + continueButton.height
+            + continueButton.anchors.topMargin + continueButton.anchors.bottomMargin
+        
```

Comment:
Extra spaces

```suggestion

```

must be removed here to make `test/lint/lint-whitespace.py` happy :)